src/components/Dashboard/TodoList.vue

```diff
@@ -94,12 +91,14 @@
                     the most common reason is an overdue annual report.
                   </p>
 
-                  <p>To resolve this issue, you MUST contact BC Registries staff:</p>
+                  <p>
+                    To resolve this issue, you MUST contact BC Registries staff:
+                  </p>
                   <ContactInfo class="mt-4 contact-info-warning" />
                 </div>
 
-                <!-- alteration in good standing -->
-                <div v-else-if="isTypeAlteration(item) && item.goodStanding && !isBenBcCccUlc && !isCoop"
+                <!-- alteration to a BEN in good standing -->
+                <div v-else-if="isTypeAlteration(item) && isAlteringToBen && isGoodStanding"
```

Comment:
This section is actually specific to alter-to-BEN filings but was incorrectly displaying for BC/CCC/ULC alterations.